.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+Linked PRs: (links to corresponding PRs, optional)
+
+## Changes
+- (changes here)
+
+## TODOs:
+- [ ] +1
+- [ ] Updated README
+- [ ] Updated CHANGELOG
+- [ ] (Other applicable TODOs)
+- [ ] (Feedback from reviewers)
```

Comment:
Agreed @jeremywiebe . This entry is here because previously in some PRs it made sense to group PR feedback into TODO items at the top of the PR so that later reviewers can see what was actioned. Since that is a process that is open to change as per specific team/project workflow, I'll remove from this template (since this one is specific to the code-style repo).